src/class.jl

```diff
@@ -0,0 +1,63 @@
+"""
+    Class
+
+A type used for controlling which classes of `TimeZone` are considered valid. Instances of
+`Class` can be combined using bitwise operators to generate a mask which allows multiple
+classes to be considered valid at once.
+
+Currently supported masks are:
+
+- `Class.FIXED`: Class indicating the time zone name is parsable as a fixed UTC offset.
+- `Class.STANDARD`: The time zone name is included in the primary IANA tz source files.
+- `Class.LEGACY`: The time zone name is included in the deprecated IANA tz source files.
+- `Class.NONE`: Mask that will match nothing.
+- `Class.DEFAULT`: Default mask used by functions: `Class.FIXED | Class.STANDARD`
+- `Class.ALL`: Mask allowing all supported classes.
+"""
+struct Class
+    val::UInt8
+end
+
+function Base.getproperty(::Type{Class}, field::Symbol)
+    if field == :NONE
```

Comment:
Might as well use `===` when comparing against symbols. The compiler likes `===`. All hail the compiler.

Reply:
Good point